core/src/main/java/org/apache/accumulo/core/conf/AccumuloConfiguration.java

```diff
@@ -209,13 +209,19 @@ static public long getMemoryInBytes(String str) {
         case 'B':
           multiplier = 0;
           break;
+        case '%':
```

Comment:
I'm wondering if it would make more sense to create a new `PropertyType` instead of piggy-backing on `MEMORY`. Is it confusing to allow percent-values for some of these other properties which we really never expect users to do?

For example, some properties like `table.file.blocksize` or `tserver.compaction.major.throughput` make no sense to allow a percentage of total heap.